src/components/atoms/UserAvatar/UserAvatar.scss

```diff
@@ -27,6 +27,7 @@
     height: 100%;
     width: 100%;
     border-radius: 50%;
+    vertical-align: unset;
```

Comment:
override the library setting

Reply:
@sunny-viktoryia 
![image](https://user-images.githubusercontent.com/63194656/124736912-235a4c80-df20-11eb-9bc6-b4fb34c7f91c.png)